scripts/monitoring/cron-send-etcd-status.sh

```diff
@@ -0,0 +1,13 @@
+#!/bin/bash
+# setting up the certificate and the key
+CERT=`ls /etc/openshift/node/system*node*.crt`
+KEY=`ls /etc/openshift/node/system*node*.key`
+
+# querying the health ping
+wget -q -O- --no-check-certificate --certificate=${CERT} --private-key=${KEY} https://0.0.0.0:10250/healthz/ping |grep -c "ok" >/dev/null
```

Comment:
This seems like it's monitoring the kubelet itself, not etcd, which is fine and needed, but this check says it's to monitor etcd specifically.

Am I missing something?